commands/completion.go

```diff
@@ -25,6 +25,10 @@ To configure your bash shell to load completions for each session, add the follo
 			if err != nil {
 				return errors.Wrap(err, "getting pack home")
 			}
+
+			if err = config.MkdirAll(packHome); err != nil {
+				return errors.Wrap(err, "creating pack home: "+packHome)
```

Comment:
Minor: We have a preference to use `Wrapf` instead of string contamination.

Reply:
Updated.